core-build/gulp-core-build/src/logging.ts

```diff
@@ -77,13 +77,13 @@ const localCache: ILocalCache = globalInstance.__loggingCache = globalInstance._
   writeSummaryCallbacks: [],
   exitCode: 0,
   writeSummaryLogs: [],
-  errorAndWarningSuppressions: {},
+  errorAndWarningSuppressions: [],
   gulp: undefined,
   gulpErrorCallback: undefined,
   gulpStopCallback: undefined,
   shouldLogErrorsDuringSummary: false,
   shouldLogWarningsDuringSummary: false
-};
+} as ILocalCache;
```

Comment:
That was a holdover. I corrected the interface so this cast wouldn't be necessary. Removing...

---
In reply to: [126799535](https://github.com/Microsoft/web-build-tools/pull/268#discussion_r126799535) [](ancestors = 126799535)